ocp_resources/node_health_check.py

```diff
@@ -8,3 +8,73 @@ class NodeHealthCheck(Resource):
     """
 
     api_group = Resource.ApiGroup.REMEDIATION_MEDIK8S_IO
+    self_node_remediation_api_group = Resource.ApiGroup.SELF_NODE_REMEDIATION_MEDIK8S_IO
+
+    def __init__(
+        self,
```

Comment:
- please use `kwargs` for anyting that is not related to the specific resource.
- The resource inherits from `Resource` yet a ns is added here

Reply:
okay